sections/related_works.tex

```diff
@@ -0,0 +1,37 @@
+\section{Related Work}
+
+\citet{mimno2008topic} describes two general topic model patterns, called ''upstream`` and ''downstream`` topic models.
+These patterns describe how metadata is incorporated in topic models.
+In the downstream pattern, metadata is incorporated in the model by having the topic model generate both the words and the metadata simultaneously.
+Here each topic has a distribution over words, as well as a distribution over metadata values.
+In the upstream pattern, the topic model is conditioned on metadata elements by representing document-topic distributions as element specific distributions.
+This way the model learns assignments of the words in each document to the metadata.
+
+\citet{author_topic_2012} have developed a \gls{lda} model called the Author-Topic model, which incorporates authorship information in the \gls{lda} model.
+Specifically, each document has a vector of authors, where for each word an author is drawn from this vector.
+The author is then used in combination with an author-topic distribution to draw a specific topic that this author writes about.
+The words from the topic-word distribution can be generated based on this topic.
+The purpose of using authorship information in this way, is to show patterns in which topics an author usually writes about, and be able to explore how related authors are in what they write about.
+\citeauthor{author_topic_2012} also show that the combination of authorship and \gls{lda} yield more coherent topics.
+
+
+\citet{MetaLDA2017} presents a resent model which can incorporate both meta data information and word embeddings within a topic model.
+Since the field of incorporating word embedding within generative topic model have gained popularity\cite{dieng2020topic}, the authors of \cite{MetaLDA2017} show how to use this information for a variety of different datasets.
```

Comment:
```suggestion
Since the field of incorporating word embeddings within generative topic models have gained popularity\cite{dieng2020topic}, \citet{MetaLDA2017} show how to use this information for a variety of different datasets.
```